spark/src/main/scala/org/apache/spark/sql/delta/SnapshotManagement.scala

```diff
@@ -29,7 +29,7 @@ import scala.util.control.NonFatal
 
 import com.databricks.spark.util.TagDefinitions.TAG_ASYNC
 import org.apache.spark.sql.delta.actions.Metadata
-import org.apache.spark.sql.delta.managedcommit.{Commit, CommitStore, GetCommitsResponse}
+import org.apache.spark.sql.delta.managedcommit.{Commit, GetCommitsResponse, TableCommitStore => CommitStore}
```

Comment:
`TableCommitStore => CommitStore` is not a good long-term solution. Can you refactor this in a follow up? Please don't do it in this one because this one is already very big.

Reply:
Yes makes sense.